ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -123,15 +123,17 @@ public boolean onInterceptTouchEvent(MotionEvent ev) {
     if (!mScrollEnabled) {
       return false;
     }
-
-    if (super.onInterceptTouchEvent(ev)) {
-      NativeGestureUtil.notifyNativeGestureStarted(this, ev);
-      ReactScrollViewHelper.emitScrollBeginDragEvent(this);
-      mDragging = true;
-      enableFpsListener();
-      return true;
+    try {
+      if (super.onInterceptTouchEvent(ev)) {
+          NativeGestureUtil.notifyNativeGestureStarted(this, ev);
+          ReactScrollViewHelper.emitScrollBeginDragEvent(this);
+          mDragging = true;
+          enableFpsListener();
+          return true;
+      }
+    } catch (Exception e) {
+        // Swallow error
```

Comment:
We should handle the error close to the place where it occurs - either prevent it from happening or log it. Never swallow exceptions.

Please use 2 spaces for indentation.